core/parameters/src/vm.rs

```diff
@@ -27,7 +27,7 @@ use std::hash::{Hash, Hasher};
 )]
 #[cfg_attr(feature = "clap", derive(clap::ValueEnum))]
 pub enum VMKind {
-    /// Wasmer 0.17.x VM.
+    /// Wasmer 0.17.x VM. Gone now.
```

Comment:
can we put the deprecated attribute on it?

Reply:
I tried that but for some reason I haven't investigated yet it would say that there's a deprecated use on the line 32 (i.e. the enum definition) and nothing I did with `#[allow(deprecated)]` would make it go away. This probably comes from a bug in one of the derives we use.

I'll eventually remove these variants altogether, so I didn't spend too much effort looking into it.